wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -58,9 +58,15 @@
   private static final Logger LOG = LoggerFactory.getLogger(WorkspaceManager.class);
 
   /** This attribute describes time when workspace was created. */
-  public static final String CREATED_ATTRIBUTE_NAME = "created";
-  /** This attribute describes time when workspace was last updated or started/stopped/recovered. */
-  public static final String UPDATED_ATTRIBUTE_NAME = "updated";
+  protected static final String CREATED_ATTRIBUTE_NAME = "created";
+  /** This attribute describes time when workspace was last updated or started */
```

Comment:
Why is it still used for storing the time when a workspace is started?

Reply:
why it should not ?

Reply:
Because it is `UPDATED` attribute but not `STARTED`? Earlier `UPDATED` was used for all actions with a workspace. Now it is used only for registering of update and start, but not for a stop. IMHO It's a quite unclear behavior.